README.md

```diff
@@ -17,7 +17,7 @@ To get a local copy up and running follow these simple example steps.
 
 - Open terminal in cloned repo directory
 
-- then run ```bundle install```.
+- then run ```ruby main.rb```.
```

Comment:
- [ ] In the README file, please fix the MIT link in order to maintain a professional repo. 🙏

Reply:
Actioned.

Reply:
This issue is well resolved! ✔️